.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const path = require('path');
+
+const prettierOptions = JSON.parse(
+  fs.readFileSync(path.resolve(__dirname, '.prettierrc'), 'utf8'),
+);
+
+module.exports = {
+  root: true,
+  parser: 'babel-eslint',
+  extends: ['plugin:jasmine/recommended', 'airbnb-base', 'prettier'],
```

Comment:
Instead of extending `prettier`, this could extend `plugin:prettier/recommended`. This basically comes down to extending `prettier`, adding the `prettier` plugin, and enabling the `prettier/prettier` rules.

Also there is no need to manually pass prettier options. `eslint-plugin-prettier` will read them for you if omitted.

Reply:
I (re)read the readme. Adding `plugin:prettier/recommended` is indeed the same as I did before. Added it now.

Reply:
Might be, that feature is not well documented. I assume it works like that indeed.